### DISCUSSION TITLE
How can I update Security Onion from version 2.3 to version 2.4?

### DISCUSSION THREAD

**Discussion by NareeratPr** [Security-Onion-Solutions/securityonion#11254] [Unsupported Versions]
Currently, I am using Security Onion version 2.3.260, which is installed on Ubuntu 20.04.6. I would like to know how to update Security Onion from version 2.3 to 2.4 while keeping all the old data and configurations.

**Comment by tomjones1977** [comment]
> You cannot do an in-place upgrade from 2.3 to 2.4. We are still investigating data migration.

https://blog.securityonion.net/2023/08/security-onion-24-has-reached-general.html

The 2.4 blog post currently says that 2.3 to 2.4 upgrade isn't possible.
